ecosystem_cicd_tools/release.py

```diff
@@ -247,3 +244,49 @@ def blueprint_release_with_latest(plugin_name,
         )
     latest_release = get_most_recent_release()
     update_latest_release_resources(latest_release)
+
+def plugin_release_with_latest(plugin_name,
+                               version=None,
+                               plugin_release_name=None,
+                               plugins=None):
+    # if we have release for this version we do not want update nothing
+    if not get_release(version):
+        # Create release for the new version if not exists
+        version_release = plugin_release(plugin_name, version,
+                                         plugin_release_name)
+        latest_release = get_release("latest")
+        if latest_release:
+            # We have latest tag and release so we need to delete
+            # them and recreate.
+            latest_release.delete_release()
+            delete_latest_tag_if_exists()
+
+        # create latest release
+        logging.info(
+            'Create release with name latest and tag latest'.format(
+                name=latest_release.tag_name))
+        plugin_release(plugin_name, "latest",
+                       plugin_release_name=version_release.body)
+        # TODO:handle assets!
+
+
+def delete_latest_tag_if_exists():
+    repo = get_repository()
+    try:
+        logging.info(
+            'Attempting  to delete Tag with name "latest" in repository {repo}.'.format(
+                repo=repo.name))
+        latest_tag_ref = repo.get_git_ref('tags/latest')
+    except UnknownObjectException:
+        logging.info(
+            'Tag with name "latest" doesnt exists.'.format(repo=repo.name))
+        return
+    latest_tag_ref.delete()
+
+
+if __name__ == '__main__':
```

Comment:
Should probably export all these in your test environment, intead of setting them inside the script.

Reply:
yea i know i will remove that . I didn't succeed doing that with pycahrm.